pkg/build/api/types.go

```diff
@@ -12,6 +12,8 @@ const (
 	BuildAnnotation = "openshift.io/build.name"
 	// BuildLabel is the key of a Pod label whose value is the Name of a Build which is run.
 	BuildLabel = "build"
+	// BuildIndicatorLabel is the key of a Pod label which can be used to query for all build pods
+	IsBuildPodLabel = "isBuildPod"
```

Comment:
can't query on annotations, right?  i do this later:

	selector := labels.LabelSelector{}.Add(buildapi.IsBuildPodLabel, labels.EqualsOperator, []string{"true"})
	podList, err := lw.client.KubeClient.Pods(kapi.NamespaceAll).List(selector, fields.Everything())